app/src/main/java/com/example/imdb/SignUpActivity.kt

```diff
@@ -13,6 +13,7 @@ import com.example.imdb.database.UserDatabase
 import com.example.imdb.databinding.ActivitySignUpBinding
 import kotlinx.coroutines.launch
 
+@Suppress("SENSELESS_COMPARISON")
```

Comment:
please do not suppress this kind of alerts

Reply:
ok, thanks!